### PR TITLE
unixcw : init at 3.5.1

### PR DESCRIPTION
**Pull request by MarcFontaine**
###### Motivation for this change

I would like to add the unixcw package to nixpkgs

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/unixcw/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{stdenv, fetchurl, libpulseaudio, alsaLib , pkgconfig, qt5}:
+stdenv.mkDerivation rec {
+  name = "unixcw-${version}";
+  version = "3.5.1";
+  src = fetchurl {
+    url = mirror://sourceforge/unixcw/unixcw_3.5.1.orig.tar.gz;
```
Comment (ryantm) [inline pkgs/applications/misc/unixcw/default.nix]: Please use ${version} in the URL so the package can be updated more easily.

File: pkgs/applications/misc/unixcw/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    sha256 ="5f3aacd8a26e16e6eff437c7ae1e9b389956fb137eeb3de24670ce05de479e7a";
+  };
+  patches = [
+    ./remove-use-of-dlopen.patch.gz
```
Comment (ryantm) [inline pkgs/applications/misc/unixcw/default.nix]: gzipping this patch seems excessive to me; how about we just have the .patch file unzipped.

**Review by ryantm — CHANGES_REQUESTED** [review]
Also please squash your commits into one commit.

**Comment by MarcFontaine** [comment]
Hi Ryan,
Thanks for looking at my pull request. I'm still new to this.
Is it possible to update a pull request with a squashed commit
or to I have to close this pull request and open a new one?

**Comment by ryantm** [comment]
@GrahamcOfBorg build unixcw

**Pull request merged**